tests/DatastreamTest.php

```diff
@@ -204,7 +206,22 @@ public function testContentsSet() {
     $this->assertEquals(3, $newds->size);
   }
 
-  public function testContentSetUrl() {
+  public function testContentSetUrlHttp() {
+    $temp = tempnam(sys_get_temp_dir(), 'tuque');
+    $this->ds->setContentFromUrl('http://www.loc.gov/standards/mods/v3/modsejournal.xml');
+    $actual = file_get_contents('http://www.loc.gov/standards/mods/v3/modsejournal.xml');
```

Comment:
Almost... want this URL in a constant, since it's referenced a number of places.